librz/core/cmd_type.c

```diff
@@ -729,6 +725,76 @@ static void set_offset_hint(RzCore *core, RzAnalysisOp *op, const char *type, ut
 	}
 }
 
+RZ_API void rz_core_list_loaded_typedefs(RzCore *core, const char *input, Sdb *TDB, RzOutputMode mode) {
+	PJ *pj = NULL;
+	if (mode == RZ_OUTPUT_MODE_JSON) {
+		pj = pj_new();
+		pj_o(pj);
+	}
+	char *name = NULL;
+	SdbKv *kv;
+	SdbListIter *iter;
+	SdbList *l = sdb_foreach_list(TDB, true);
+	ls_foreach (l, iter, kv) {
+		if (!strcmp(sdbkv_value(kv), "typedef")) {
+			if (!name || strcmp(sdbkv_value(kv), name)) {
+				free(name);
+				name = strdup(sdbkv_key(kv));
+				if (!input[1]) {
+					rz_cons_println(name);
+				} else {
+					const char *q = sdb_fmt("typedef.%s", name);
+					const char *res = sdb_const_get(TDB, q, 0);
+					pj_ks(pj, name, res);
+				}
+			}
+		}
+	}
+	if (mode == RZ_OUTPUT_MODE_JSON) {
+		pj_end(pj);
+	}
+	if (pj) {
+		rz_cons_printf("%s\n", pj_string(pj));
+		pj_free(pj);
+	}
+	free(name);
+	ls_free(l);
+}
+
+RZ_API void rz_core_list_typename_alias_c(RzCore *core, const char *input, Sdb *TDB) {
```

Comment:
Instead of `input` change name to something more meaningful, like `typedef_name`